LibraryApp/src/components/form.js

```diff
@@ -97,36 +127,24 @@ class Form extends React.PureComponent {
   }
 
   handleFormSubmit() {
+    const {updateBooks} = this.props;
     const data = {...this.state};
     this.setState({isLoading: true});
-    fetch(user, {
-      method: 'POST',
-      headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify(data),
-    })
-      .then((rawResponse) => rawResponse.json())
-      .then((response) => {
-        this.onSuccess(response);
-      })
-      .catch((error) => this.onError(error));
+    const {booksListItem, bookAllDetails} = GetBookDetails(data);
+    updateBooks(booksListItem);
+    this.onSuccess(bookAllDetails);
   }
 
-  onSuccess(response) {
+  onSuccess(bookData) {
+    const {setActiveTab} = this.props;
     this.resetAllFields();
-    alert('Success', stringifyResponse(response));
-  }
-
-  onError(error) {
-    this.setState({isLoading: false});
-    alert('Error', error.message);
+    alert('Book Details added / updated', stringifyResponse(bookData));
```

Comment:
Dont stringify response in message

Reply:
Removed bookData. Not displaying submitted book details on alert.